src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs

```diff
@@ -44,14 +44,18 @@ public class DocumentBuilder : IDisposable
             IEnumerable<Assembly> assemblies,
             ImmutableArray<string> postProcessorNames,
             string templateHash,
-            string intermediateFolder = null)
+            string intermediateFolder = null,
+            string commitFromSHA = null,
```

Comment:
in camel casing should use `Sha`? 
[http://dotnet.github.io/docfx/guideline/csharp_coding_standards.html#naming-mandatory](http://dotnet.github.io/docfx/guideline/csharp_coding_standards.html#naming-mandatory)

---
In reply to: [85288594](https://github.com/dotnet/docfx/pull/831#discussion_r85288594) [](ancestors = 85288594)